lms/templates/static_templates/ferie_bez_nudy.html

```diff
@@ -0,0 +1,549 @@
+<%namespace name='static' file='../static_content.html'/>
+<%page expression_filter="h"/>
+<%! from django.utils.translation import ugettext as _ %>
+<%inherit file="../main.html" />
+
+<%block name="pagetitle">${_("#FerieBezNudy")}</%block>
+
+<%block name="og_meta">
+## OG (Open Graph) title and description added below to give social media info to display
+## (https://developers.facebook.com/docs/opengraph/howtos/maximizing-distribution-media-content#tags)
+<meta property="og:title" content="${_('#FerieBezNudy')}" />
+<meta property="og:description" content="${_('CHANGE_ME')}" />
```

Comment:
pewnie będzie to shareowane więc warto ustawić dla fb; można przemyśleć i dodać name="twitter:"  i name="thumbnail" dla google

Reply:
tak, to pominięto, nie ustalono co z sharowaniem na social media.

Reply:
ustalenia:
og:title "Navoica dla młodzieży"
og:description "Webinary dla uczniów - programowanie i prawo"
og:image wyrzucamy, domyślny będzie, zobaczymy jak to wskoczy

Reply:
Zaktualizowano